src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main_header">
+      <a
+        href="/"
+        class="logo_img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav>
+        <ul class="device_nav nav__list">
+          <li class="nav__item">
+            <a
+              class="device_category nav__link is-active"
```

Comment:
The class 'device_category' is unnecessary since 'nav__link' already serves the purpose of styling navigation links. Use meaningful class names that represent the content.